data-prep/README.md

```diff
@@ -15,9 +15,59 @@ Data Cleaning and Munging
 
 A number of cleaning steps are necessary to use the raw data.
 
-### Project Filtering
+### Consumption Data Cleaning
+
+**Elec Traces**
+
+* Remove duplicates
+  * No duplicates in monthly data
+* Add leading zeroes to `PREM_ID` where missing (length == 8 or 9) in monthly traces.
+
+* 7702 elec monthly traces with unique SA_ID
+
+**Gas Traces**
+
+* Remove duplicates
+  * No duplicates in monthly data
+* Add leading zeroes to `PREM_ID` where missing (length == 8 or 9) in monthly traces.
+
+* 6919 gas monthly traces with unique SA_ID
+
+**Elec Xrefs**
+
+* Add leading zeroes to `char_prem_id` where missing (length == 8 or 9). Appear to have been lost during PGE's export process 
+
+**For Caltrack data analysis: traces with `is_net_metered=Y` should be excluded from analysis.**
```

Comment:
Looks like the full explanation was chopped out. From the original: https://github.com/impactlab/caltrack-betatest/blob/4ad1223a4d3d10e6e0e11b1ec9337fac68c11391/data-prep/data-cleaning-working-group.md


* Add `is_net_metered` column
  * Collect all SAIDs from 15min and hourly traces that have `DIR==R`
  * Map these SAIDs to corresponding `char_prem_id`
  * Mark crossrefs that match any of the found `char_prem_id`s OR have `net_mtr_ind==Y` as `is_net_metered=Y`
  * 1905 crossrefs marked as net metered, (688 with unique `char_prem_id`)